F/FMM3D/build_tarballs.jl

```diff
@@ -16,75 +16,66 @@
 using BinaryBuilder, Pkg
 
 name = "FMM3D"
-version = v"0.1.0"
+version = v"1.0.1"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/flatironinstitute/FMM3D.git", "e42473a8091d33a83cbdb631cff4660ce7f94a96")
+    GitSource("https://github.com/flatironinstitute/FMM3D.git", "9228240c090d1b8386617728ffa14372fe967b1a")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir
-cd FMM3D/
+cd $WORKSPACE/srcdir/FMM3D/
 
-touch make.inc
-
-# clang has some issue with openmp (lomp not found)
-# force gcc
+OMPFLAGS="-fopenmp"
+OMPLIBS="-lgomp"
+# openmp library name is different on bsd systems
 if [[ ${target} = *apple* || ${target} = *freebsd* ]]; then
-  export CC="gcc"
+  export OMPLIBS="-lomp"
 fi
-
-echo "CC=${CC}" >> make.inc;
-echo "CXX=${CXX}" >> make.inc;
-echo "FC=${FC}" >> make.inc;
-
+FFLAGS="-fPIC -O3 -funroll-loops -std=legacy"
 if [[ ${target} = *mingw* ]]; then
-    echo "FFLAGS= -fPIC -O3 -fno-asynchronous-unwind-tables -funroll-loops" >> make.inc;
-else
-    echo "FFLAGS= -fPIC -O3 -funroll-loops" >> make.inc;
+    FFLAGS="${FFLAGS} -fno-asynchronous-unwind-tables"
 fi
+LIBS="-lm"
```

Comment:
Needs to bring libgfortran now that you're linking with `cc`:
```suggestion
LIBS="-lm -lgfortran"
```

Reply:
Agreed.